blocks_vertical/looks.js

```diff
@@ -281,6 +298,49 @@ Blockly.Blocks['looks_size'] = {
   }
 };
 
+Blockly.Blocks['looks_changestretchby'] = {
+  /**
+   * Block to change stretch. Does not actually do anything. This is an
+   * obsolete block that is implemented for compatibility with Scratch 2.0
+   * projects.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      "message0": Blockly.Msg.LOOKS_CHANGESTRETCHBY,
+      "args0": [
+        {
+          "type": "input_value",
+          "name": "CHANGE"
+        }
+      ],
+      "category": Blockly.Categories.looks,
+      "extensions": ["colours_looks", "shape_statement"]
+    });
+  }
+};
+
+Blockly.Blocks['looks_setstretchto'] = {
+  /**
+   * Block to set stretch. Does not actually do anything. This is an obsolete
+   * block that is implemented for compatibility with Scratch 2.0 projects.
```

Comment:
This is not exactly true :)

I say "Scratch 2.0 projects", but really it's very very old pre-1.4 projects that had the `setStretchTo` and `changeStretchBy` blocks. [They don't exist in the scratch-flash source code](https://github.com/LLK/scratch-flash/blob/develop/src/Specs.as), and there are [projects that use them from 2010](https://scratch.mit.edu/projects/1119715/). (The Scratch wiki [implies](https://en.scratch-wiki.info/wiki/Set_Stretch_to_()_(block)) they existed in 1.4 (though nonfunctional) and were removed altogether in 2.0.)

Reply:
Thanks for the comment and the research. =] Can you add a mention for 1.4 projects here then? Or feel free to change all of these comments to say something like `for compatibility with legacy blocks` instead.

Reply:
Okay, there's a comment here explaining this now :)